### ISSUE TITLE
Scoreboard Data from older years

### ISSUE THREAD

**Issue by adamdiel** [mkreiser/ESPN-Fantasy-Football-API#145]
So I read all about how ESPN deleted box scores from anything older than 1 year old. 

The final scores are still available in the scoreboard. 

Looking through the history of the codebase, I saw that their use to be a scoreboard class. Curious if that is now broken or if there are plans on bringing this functionality back?

**Comment by adamdiel** [comment]
So wanted to make it clear I have never worked with node and limited with my API knowledge so I apologize if I say anything too silly. 

I spent a few hours last night on Chrome figuring out what the API call might look like. 

I was able to get this one working that gave me the scores of every game of the season 

```
curl --cookie "espn_s2=REDACTED;SWID={REDACTED}" 'https://fantasy.espn.com/apis/v3/games/ffl/leagueHistory/LEAGUEIDREDACTED?scoringPeriodId=9&view=mMatchupScore&view=mScoreboard&view=mSettings&view=mTopPerformers&view=mTeam&seasonId=2017'
```

**Comment by mkreiser** [comment]
@adamdiel Good catch. For whatever reason, I did not realize this. I added `getHistoricalScoreboardForWeek` to the client in #146 and published the code as `0.14.0` for use.

**Comment by adamdiel** [comment]
Thank you for adding this! You are awesome!
